src/main/java/mate/academy/internetshop/controller/AddItemToBucketController.java

```diff
@@ -10,6 +10,7 @@
 import mate.academy.internetshop.model.Item;
 import mate.academy.internetshop.service.BucketService;
 import mate.academy.internetshop.service.ItemService;
+import mate.academy.internetshop.service.UserService;
 
 public class AddItemToBucketController extends HttpServlet {
     private static final Long USER_ID = 1L;
```

Comment:
we can remove this constant

Reply:
> how about changing CompleteOrderController, DeleteItemFromBucketController, ShowOrdersController?

It`s a task for the second part of home work

Reply:
> we can remove this constant
I`ve fixed it